it/uk/gov/hmrc/claimvatenrolmentfrontend/stubs/AuthStub.scala

```diff
@@ -42,7 +42,7 @@ trait AuthStub extends WireMockMethods {
     "internalId" -> internalId
   )
 
-  def successfulAuthResponseEnrolmentAllocation(groupId: Option[String], internalId: Option[String]): JsObject = Json.obj(
+  def successfulAuthResponse(groupId: Option[String], internalId: Option[String]): JsObject = Json.obj(
```

Comment:
This was changed to an overloaded method as the previous name didn't really make sense